etc/allow-lua.inc

```diff
@@ -4,5 +4,6 @@ include allow-lua.local
 
 noblacklist ${PATH}/lua*
 noblacklist /usr/include/lua*
+noblacklist /usr/lib/liblua*
```

Comment:
Where is the blacklist? `grep "/usr/lib/liblua" etc/*.inc`

Reply:
@rusty-snake I had that in my disable-interpreters.local and forgot about it. Thanks for the catch!